<launch>
    <node pkg="duo3d_ros" name="duo_node" type="duo_node" required="true" output="screen">
		<!-- Supported Device Names: [Mini, Mini-lx] -->
		<param name="device_name" 		value="Mini-lx" />
		<param name="device_serial_number" 	value="N-A" />

    	<param name="frame_id"   value="duo3d_camera" />
		<!-- 	Choose anywhere from 0.1-3000 Frames Per Second (FPS) for the DUO to publish at. Please only
		use up to one decimal place ONLY, if you need to. This does not require a specific rate, like
		the duo_dense3d.launch file requires.

		 56 FPS @ 752x480 ->  49
		 62 FPS @ 640x480
		123 FPS @ 640x240
		240 FPS @ 640x120
		 93 FPS @ 320x480
		184 FPS @ 320x240 -> 171
		360 FPS @ 320x120

		-->
		<param name="FPS"   value="35" />

		<param name="resolution_width"	value="752" />
		<param name="resolution_height"	value="480" />

		<param name="exposure"		value="100"  />
		<param name="gain"		value="20"  />
		<param name="led_lighting"	value="100"  />

	    <param name="use_DUO_imu"   	value="true" />
	    <param name="use_DUO_LEDs"   	value="true" />
      <param name="publishDepth"   	value="true" />
      <param name="publishDepthImage"   	value="true" />


    </node>


</launch>
